README.md

```diff
@@ -43,7 +43,7 @@ Refer to [torchserve docker](docker/README.md) for details.
   * [Kubeflow](https://v0-5.kubeflow.org/docs/components/pytorchserving/)
   * [MLflow](https://github.com/mlflow/mlflow-torchserve)
   * [Sagemaker](https://aws.amazon.com/blogs/machine-learning/serving-pytorch-models-in-production-with-the-amazon-sagemaker-native-torchserve-integration/)
-  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/)
+  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/) (Supports both v1 and v2 api)
```

Comment:
How about this instead

```suggestion
  * [Kserve](https://kserve.github.io/website/modelserving/v1beta1/torchserve/) : Supports both v1 and v2 API
```

Reply:
Ok one last thing here seeing as how we're only linking to v1beta1 we should also link to v2 here 

`Kserve: Supports both [v1](v_1_link) and [v2](v_2_link) API